test/gerkin/features/steps/data_read_value.js

```diff
@@ -0,0 +1,97 @@
+const pactum = require('pactum');
+const { Given, When, Then, Before, After } = require('@cucumber/cucumber');
+const { header } = require('./helpers/helpers');
+
+let searchedRecord;
+let specDataReadValue = pactum.spec();
+
+const baseUrl = uuid =>
+  `http://localhost:3333/data/registry1/version1/${uuid}/read-value/User.FirstName`;
```

Comment:
In other place baseUrl is string, and in this context it's function used to build URL.  
Additioanlly User.FirstName are separate url parameters which is not clear from this context.  

Can you change it to function which takes uuid, field and ext as parameters and then builds desired URL string? 

Additionally, `baseUrl` should be configured from config file or from environmental variable.
Adjustments on baseUrl should be done only in one place not across multiple test cases.
While it's fine to make some assumptions about state of the application (e.g. existing users), I don't think all apps will work on `localhost:3333`. Can you move `http://localhost:3333/` part to some global constant? 

Partially, this brings us back to the discussion regarding example apps setup. At the next tech meet we should definitely start a discussion on what can be configured and what we can expect from the test instances. 
cc: @dszafranek

Reply:
Good idea!
I will change url and provide all 3 parameters as variables and move `http://localhost:3333/` to global constant